code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -166,11 +166,14 @@
 		if(ismob(G.grabbed_thing))
 			var/mob/M = G.grabbed_thing
 			var/atom/blocker = LinkBlocked(user, user.loc, loc)
+			if(!src.Adjacent(M))
+				visible_message(SPAN_DANGER("[M] is too far to place onto the [src]"))
```

Comment:
Referencing an atom in a string will automatically add "the" before it if its name is an improper noun, so the "the" isn't needed. (also, needs a full stop/period)
**Before:**
![before](https://github.com/cmss13-devs/cmss13/assets/57483089/c69566d0-9962-487f-9950-30921be21807)
**After:**
![after](https://github.com/cmss13-devs/cmss13/assets/57483089/9cfafe98-013d-4068-826f-8b5ccffdf33b)
```suggestion
				visible_message(SPAN_DANGER("[M] is too far to place onto [src]."))
```